cosmos/carbon.json

```diff
@@ -143,6 +143,50 @@
         "high": 115384615.38461539
       }
     },
+    {
+      "coinDenom": "STARS",
+      "coinMinimalDenom": "ibc/07FA7831E1920D0C87C9388F86B0108677F6ED0C9DE7E4063F05ED675192405C",
+      "coinDecimals": 6,
+      "coinGeckoId": "stargaze",
+      "gasPriceStep": {
+        "low": 2.6923076923076925,
+        "average": 3.365384615384616,
+        "high": 4.038461538461538
+      }
+    },
+    {
+      "coinDenom": "LUNA",
+      "coinMinimalDenom": "ibc/2B58B8C147E8718EECCB3713271DF46DEE8A3A00A27242628604E31C2F370EF5",
+      "coinDecimals": 6,
+      "coinGeckoId": "terra-luna-2",
+      "gasPriceStep": {
+        "low": 0.038461538461538464,
+        "average": 0.04807692307692308,
+        "high": 0.057692307692307696
+      }
+    },
+    {
+      "coinDenom": "STRD",
+      "coinMinimalDenom": "ibc/3552CECB7BCE1891DB6070D37EC6E954C972B1400141308FCD85FD148BD06DE5",
+      "coinDecimals": 6,
+      "coinGeckoId": "stride",
+      "gasPriceStep": {
+        "low": 0.2461538461538462,
+        "average": 0.30769230769230776,
+        "high": 0.3692307692307693
+      }
+    },
+    {
+      "coinDenom": "KUJI",
+      "coinMinimalDenom": "ibc/662914D0C1CEBCB070C68F061D035E8B10A07C79AB286E7342C85F3BE74612C5",
+      "coinDecimals": 6,
+      "coinGeckoId": "kujira",
+      "gasPriceStep": {
+        "low": 0.11538461538461538,
+        "average": 0.14423076923076922,
+        "high": 0.17307692307692307
```

Comment:
For this currency, the low step is not high enough, i.e.
(0.11538461538461538*130000)/10^6 ~ 0.0149 KUJI
<img width="519" alt="Screenshot 2023-02-09 at 7 26 22 PM" src="https://user-images.githubusercontent.com/65583522/217800084-2d5ab8ad-8c3e-4a0e-85d1-4fcfea9f5e35.png">

Probably need to increase low step slightly, e.g. `0.11538461538461539`, and then adjust average (x1.25) and high step (x1.5) accordingly.